tests/utils/test_graph2.py

```diff
@@ -0,0 +1,54 @@
+import os.path
+
+import pytest
+import requests
+from pathvalidate import sanitize_filename
+
+from quark.core.apkinfo import AndroguardImp as Apkinfo
+from quark.utils.graph import call_graph
+
+
+@pytest.fixture(scope="module")
+def analysis_object(tmp_path_factory):
+    APK_SOURCE = (
+        "https://github.com/quark-engine/" "apk-samples/raw/master/test-samples/F84E990228E08B52CDB839FB10BA36DFC187945CE94D73EB64916D453D95DA57.apk"
+    )
+    APK_NAME = "F84E990228E08B52CDB839FB10BA36DFC187945CE94D73EB64916D453D95DA57.apk"
+
+    # Download apk
+    request = requests.get(APK_SOURCE)
+    apk_file = tmp_path_factory.getbasetemp() / APK_NAME
+    apk_file.write_bytes(request.content)
+    analysis = Apkinfo(apk_file)
+
+    yield analysis
+
```

Comment:
This method is very similar to the `analysis_object` method in `test_graph.py`. The only difference is the values of `APK_SOURCE` and `APK_NAME`. Hence, I recommend using Pytest's [fixture-parametrizing feature](https://docs.pytest.org/en/stable/how-to/fixtures.html#parametrizing-fixtures) to avoid code duplication, as we did with [the `apkinfo_with_R2Imp_only` method in `test_apkinfo.py`](https://github.com/sidra-asa/quark-engine/blob/8dfcda3c7ddfda4761bcab117589f0de578c43cc/tests/core/test_apkinfo.py#L63).